lib/user_history.rb

```diff
@@ -0,0 +1,52 @@
+module UserHistory
+  def log_assignment_of_next_of_kin(user, kin, admin:)
```

Comment:
If admin is a required argument, is the `:` needed?

Reply:
I think the `:` is to mark `admin` as a (required) keyword argument, rather than a positional one: https://docs.ruby-lang.org/en/master/syntax/methods_rdoc.html#label-Keyword+Arguments

Reply:
Oh! TIL.